omx.h

```diff
@@ -183,11 +201,11 @@ class cOmx : public cThread
 	void SetBufferStallThreshold(int delayMs);
 	bool IsBufferStall(void);
 
-	static void OnBufferEmpty(void *instance, COMPONENT_T *comp);
-	static void OnPortSettingsChanged(void *instance, COMPONENT_T *comp, OMX_U32 data);
-	static void OnEndOfStream(void *instance, COMPONENT_T *comp, OMX_U32 data);
-	static void OnError(void *instance, COMPONENT_T *comp, OMX_U32 data);
-	static void OnConfigChanged(void *instance, COMPONENT_T *comp, OMX_U32 data);
+	static void OnBufferEmpty(void *This, COMPONENT_T *comp);
```

Comment:
what's wrong with `instance`?

Reply:
I see - however, `This` looks a bit like misbehaved auto correction to me. What's about `void *omx`?

Reply:
I can apply that, or I can retain the name `instance`. Which one would you prefer?

Reply:
then let's keep it an `instance`.